process-dir.js

```diff
@@ -135,55 +147,138 @@ async function replaceToCJS (dir, file, source) {
   await writeFile(join(dir, file.replace(/\.js$/, '.cjs')), cjs)
 }
 
-async function replacePackage (dir, file, files) {
-  let pkgFile = join(dir, dirname(file), 'package.json')
+const hasDotEnv = async pathToCurrentFile => {
+  let buffer = await readFile(pathToCurrentFile)
+  let string = buffer.toString()
+  if (string.includes(ENV_PROD) || string.includes(ENV_NOT_PROD)) {
+    return true
+  }
+  return false
+}
+
+const replaceAllFallback = (str, find, replace) => {
+  // eslint-disable-next-line security/detect-non-literal-regexp
+  return str.replace(new RegExp(find, 'g'), replace)
+}
+
+const processDotEnv = async pathToCurrentFile => {
+  let pathToProd = pathToCurrentFile.replace(/\.browser.js$/, '.prod.js')
+  let pathToDev = pathToCurrentFile.replace(/\.browser.js$/, '.dev.js')
+
+  let buffer = await readFile(pathToCurrentFile)
+  let string = buffer.toString()
+
+  let isProd = string.includes(ENV_PROD)
+  let isNotProd = string.includes(ENV_NOT_PROD)
+
+  let prodModifiedToTrue = replaceAllFallback(string, ENV_PROD, 'true')
```

Comment:
You can’t use `string` → `string` replacement for code. At least RegExp.

Reply:
Why? Is this bad practice?

Reply:
Because somebody can have `process.env.NODE_ENV=="development"` instead of `process.env.NODE_ENV !== 'production'`